ReactCommon/react/renderer/uimanager/UIManagerBinding.cpp

```diff
@@ -576,6 +576,22 @@ jsi::Value UIManagerBinding::get(
             jsi::Value const *arguments,
             size_t /*count*/) noexcept -> jsi::Value {
           auto shadowNode = shadowNodeFromValue(runtime, arguments[0]);
+          bool turboModuleCalled = false;
+          auto nativeMeasurerValue = runtime.global().getProperty(runtime, "__turboModuleProxy")
+                  .asObject(runtime).asFunction(runtime).call(runtime, "NativeFabricMeasurerTurboModule");
+
+          if (nativeMeasurerValue.isObject()) {
```

Comment:
This check will return false for Paper?

Reply:
Good question! Luckily this entire file is only used for Fabric, so we don't need to check for Paper anywhere here.